static/styles/reactions.css

```diff
@@ -80,7 +80,7 @@
             color: hsl(200, 100%, 40%);
         }
 
-        &:only-child {
+        &:first-child {
```

Comment:
Merged this, since it seems correct and helpful either way.  Though actually, I'll add a comment about this subtlety.